apis/xnetworks/definition.yaml

```diff
@@ -7,6 +7,7 @@ spec:
   names:
     kind: XNetwork
     plural: xnetworks
+  scope: Namespaced
```

Comment:
For networks i personally would choose cluster-scoped tbh

Reply:
In a V2 architecture I'd say that all compositions should be namespaced unless it's absolutely necessary to make them cluster scoped. This allows for arbitrary versions to be installed for individual customers.